frontend/src/component/pages/HomePage.tsx

```diff
@@ -391,6 +395,7 @@ export class HomePage extends Component<Props, State> {
         <ChangeLogModal
           ref={this.changeLogModalRef}
           changeLog={this.state.changeLog}
+          onClose={this.handleChangeLogClose}
```

Comment:
Not on close. What happens if the user directly closes the web browser? Let's invoke `viewChangeLog` when the modal is opened.